### ISSUE TITLE
ProperEscapingFunction: account for T_OPEN_TAG_WITH_ECHO

### ISSUE THREAD

**Issue by rebeccahum** [Automattic/VIP-Coding-Standards#671]
## Bug Description

Introduced in #668, we should also account for `T_OPEN_TAG_WITH_ECHO`:

```php
<?= '<h2>' , esc_attr( $test ) , '</h2>'; // Warning.
```

However, it's not as simple as just adding it in an `OR` statement to:

https://github.com/Automattic/VIP-Coding-Standards/blob/7decb5b1f0570c69b66007394952d970643d073c/WordPressVIPMinimum/Sniffs/Security/ProperEscapingFunctionSniff.php#L111

This is because `findStartOfStatement()` will not recognize `<?=` as the start of the statement if there is inline HTML before it, e.g.

```php
<h1>TEST</h1>

<?= '<h2>' , esc_attr( $test ) , '</h2>'; // Warning.
```

However, using just the below will return the expected `$start_of_statement`:

```php
<?= '<h2>' , esc_attr( $test ) , '</h2>'; // Warning.
```

## Error Code

WordPressVIPMinimum.Security.ProperEscapingFunction.notAttrEscAttr
